scripts/gen_vimdoc.py

```diff
@@ -786,6 +796,18 @@ def has_nonexcluded_params(m):
     for child in parent.childNodes:
         para, _ = para_as_map(child, indent, width, fmt_vimhelp)
 
+        # 'programlisting' blocks are Markdown code blocks. Do not include
+        # these as a separate paragraph, but append to the last non-empty line
```

Comment:
nice! this was also a problem for `<pre>` (rendered as too much whitespace / blank lines). But we should probably get rid of all `<pre>` (separate PR) so doesn't matter.